@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -1423,8 +1403,20 @@ export class TileGeometryCreator {
                 displacementMap
             };
             object.userData = tileDisplacementMap;
-        } else {
-            object.userData = srcGeometry.objInfos;
+        } else if (
+            (hasObjInfos || isCirclesTechnique(technique) || isSquaresTechnique(technique)) &&
```

Comment:
I am wondering if there is no hidden bug here, because we may pass `undefined` srcGeometry.objInfos to `object.userData'. Of course I saw that is the part of original implementation. If I have right then entire function should return immediately if there is no object infos.

Reply:
yes, you're probably right. I didn't want to change the logic to avoid breaking something, but I really cannot think of why would that special behavior be needed for circles and squares techniques. I guess I'll try and see what happens.